src/registry.json

```diff
@@ -829,6 +829,36 @@
         }
       }
     },
+    "npm:@clustersxyz/metamask-snap": {
+      "id": "npm:@clustersxyz/metamask-snap",
+      "metadata": {
+        "name": "Clusters",
+        "author": {
+          "name": "Clusters",
+          "website": "https://clusters.xyz"
+        },
+        "website": "https://clusters.xyz",
+        "summary": "View clusters names natively within your wallet",
+        "description": "After installing the Snap, configure your cluster at https://clusters.xyz then use cluster names for sending and receiving native tokens, ERC20s, or NFTs",
+        "category": "interoperability",
```

Comment:
Remove this for now

Reply:
@Montoya used "name resolution" as category as discussed elsewhere